cluster/gce/gci/configure-helper.sh

```diff
@@ -1010,6 +1010,21 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+
+  # Create the ABAC file only if it's explicitly requested.
+  if [[ -n "${ENABLE_LEGACY_ABAC}" ]]; then
```

Comment:
I'm fine with that (and we can have the deprecation policy discussion in that issue).